src/components/ToolTip.tsx

```diff
@@ -89,8 +89,10 @@ export default function ({
   const { xs } = useBreakpoints()
   useClickOutside(childrenRef, () => setIsShow(false))
 
+  const [node, setNode] = useState<any>(null)
```

Comment:
There's a type for `node`: https://stackoverflow.com/questions/45519567/what-is-the-typescript-equivalent-of-react-proptypes-node
Don't use `any`

Reply:
Changed